srcpkgs/mozjs91/template

```diff
@@ -0,0 +1,70 @@
+# Template file for 'mozjs91'
+pkgname=mozjs91
+version=91.5.0
```

Comment:
```suggestion
version=91.7.0
```

Reply:
Latest is `91.7.1`, so I set `version=91.7.1`